### PR TITLE
Fix accessibility reported by pa11y

### PR DESCRIPTION
**Pull request by senthil10**
These are accessibility issues reported by pa11y. These are not raised by wave, none of this have any affect visually or functionally, but it is a better change code wise.

Issued addressed:

* Duplicate id in same page
* Missing `alt` tags
* Missing title for iframes

### CODE REVIEW

File: layouts/partials/navbar.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="container">
     <!-- Logo & Subtitle -->
     <div class="d-flex flex-column logo">
-      <a href="/"><img src="/img/logos/sll_dp_outline.svg" aria-label="SciLifeLab Data Platform" ></a>
+      <a href="/"><img src="/img/logos/sll_dp_outline.svg" alt="SLL DP Logo" aria-label="SciLifeLab Data Platform" ></a>
```
[the remark this answers is not in the document]
Comment (senthil10) [reply layouts/partials/navbar.html:5]: If there is an `aria-label` attribute exists, screen readers will use it, that is why `wave` did not complain about this.

Anyway will fix it while fixing other texts ^^

[the remark this answers is not in the document]
Comment (Ziip-dev) [reply layouts/partials/navbar.html:5]: Ah right, this takes precedence. No issue then :)

**Comment by senthil10** [comment]
@Ziip-dev I have fixed the text, note the `PlumX` is a tool not abbreviation, it is mentioned in the text above, so guess we can't do much there.

Also, I have fixed what was bugging me before, the redundant `img` tags, have fixed that in `css`

**Review by Ziip-dev — APPROVED** [review]
Perfect work! :ok_hand:

**Pull request merged**